server/src/main/java/org/opensearch/cluster/decommission/DecommissionAttributeMetadata.java

```diff
@@ -152,6 +164,7 @@ public Version getMinimalSupportedVersion() {
     public DecommissionAttributeMetadata(StreamInput in) throws IOException {
         this.decommissionAttribute = new DecommissionAttribute(in);
         this.status = DecommissionStatus.fromString(in.readString());
+        this.requestID = in.readString();
```

Comment:
we need to add versioning checks to ensure mixed cluster versions are able to read off from new cluster state

Reply:
+1 as Decommissioning feature has been launched in previous versions. btw did we mark it Experimental or GA?

Reply:
Decommission feature is marked experimental. Hence, should be fine to have breaking changes?

Reply:
yes in this case it would be acceptable.